endpoints/api/manifest.py

```diff
@@ -107,7 +110,14 @@ class RepositoryManifest(RepositoryParamResource):
     @require_repo_read(allow_for_superuser=True)
     @nickname("getRepoManifest")
     @disallow_for_app_repositories
-    def get(self, namespace_name, repository_name, manifestref):
+    @parse_args()
```

Comment:
could we have getting the model card info a seperate API call which the frontend can determine based on the artifactType?